### ISSUE TITLE
_dumbwinproc, _win32stdio, and _pollingfile fail to import when running tests under Windows

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8609]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8609|
|Type|enhancement|
|Created|2016-07-16 21:23:45Z|
|Branch|https://github.com/twisted/twisted/tree/8609-rodrigc-win|

Trying to run the tests with Python 3.5 and tox:

`tox -e py35-tests-windows`

results in this error:

```
===============================================================================
[ERROR]
Traceback (most recent call last):
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\trial\runner.py", line 508, in loadPackage
    module = modinfo.load()
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\python\modules.py", line 392, in load
    return self.pathEntry.pythonPath.moduleLoader(self.name)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\python\reflect.py", line 300, in namedAny
    topLevelPackage = _importAndCheckStack(trialname)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\python\reflect.py", line 247, in _importAndCheckStack
    reraise(excValue, excTraceback)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\test\test_newtls.py", line 14, in <module>
    from twisted.internet.test.test_tls import SSLCreator, TLSMixin
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\test\test_tls.py", line 21, in <module>
    from twisted.internet.endpoints import (
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\endpoints.py", line 34, in <module>
    from twisted.internet.stdio import StandardIO, PipeAddress
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\stdio.py", line 30, in <module>
    from twisted.internet import _win32stdio
builtins.ImportError: cannot import name '_win32stdio'
twisted.internet.test.test_newtls
===============================================================================
[ERROR]
Traceback (most recent call last):
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\test\test_process.py", line 662, in test_childConnectionLost
    self.runReactor(reactor)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\test\reactormixins.py", line 317, in runReactor
    "reactor still running after %s seconds" % (timeout,))
twisted.internet.test.reactormixins.TestTimeoutError: reactor still running after 120.0 seconds
twisted.internet.test.test_process.ProcessTestsBuilder_SelectReactorTests.test_childConnectionLost
===============================================================================
[ERROR]
Traceback (most recent call last):
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\base.py", line 430, in _continueFiring
    callable(*args, **kwargs)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\posixbase.py", line 352, in spawnProcess
    from twisted.internet._dumbwin32proc import Process
builtins.ImportError: No module named 'twisted.internet._dumbwin32proc'
twisted.internet.test.test_process.ProcessTestsBuilder_SelectReactorTests.test_processCommandLineArguments
twisted.internet.test.test_process.ProcessTestsBuilder_SelectReactorTests.test_systemCallUninterruptedByChildExit
twisted.internet.test.test_process.ProcessTestsBuilder_SelectReactorTests.test_systemCallUninterruptedByChildExit
twisted.internet.test.test_process.ProcessTestsBuilder_SelectReactorTests.test_systemCallUninterruptedByChildExit
twisted.internet.test.test_protocol.AdapterTests.test_consumerToProtocol
===============================================================================
[ERROR]
Traceback (most recent call last):
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\int

lugins
    allDropins = getCache(package)
--- <exception caught here> ---
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\plugin.py", line 171, in getCache
    provider = pluginModule.load()
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\python\modules.py", line 392, in load
    return self.pathEntry.pythonPath.moduleLoader(self.name)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\python\reflect.py", line 300, in namedAny
    topLevelPackage = _importAndCheckStack(trialname)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\python\reflect.py", line 247, in _importAndCheckStack
    reraise(excValue, excTraceback)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\python\compat.py", line 409, in reraise
    raise exception.with_traceback(traceback)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\plugins\twisted_core.py", line 6, in <module>
    from twisted.internet.endpoints import (
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\endpoints.py", line 34, in <module>
    from twisted.internet.stdio import StandardIO, PipeAddress
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\stdio.py", line 30, in <module>
    from twisted.internet import _win32stdio
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\internet\_win32stdio.py", line 15, in <module>
    from twisted.internet import _pollingfile, main
builtins.ImportError: cannot import name '_pollingfile'
Unexpected error while writing cache file
Traceback (most recent call last):
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\python\usage.py", line 411, in <lambda>
    fn = lambda name, value, m=method: m(value)
  File "c:\projects\twisted\build\py35-tests-windows\lib\site-packages\twisted\application\app.py", line 511, in opt_reacto
```

<details><summary>Searchable metadata</summary>

```
trac-id__8609 8609
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8609_rodrigc_win 8609-rodrigc-win
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468704225496507 1468704225496507
changetime__1468713621862448 1468713621862448
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/341

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_3"></a>|@markrwilliams commented|
|-|-|

This looks good. I have no idea how this could be modified to improve coverage without adding unrelated noise.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset fa8f3cda35c0f68ec9df266b409d8fd8af45ddf4
```
#!CommitTicketReference repository="" revision="fa8f3cda35c0f68ec9df266b409d8fd8af45ddf4"
Merge pull request #341 from twisted/8609-rodrigc-win

Author: rodrigc
Reviewer: markrwilliams
Fixes: #8609

Add _dumbwin32proc, _pollingfile, _win32stdio to fix importing on Python 3
```
